### ISSUE TITLE
Can't start addon on a new install

### ISSUE THREAD

**Issue by Tommatheussen** [hassio-addons/addon-influxdb#80]
# Problem/Motivation
The addon does not start on a new install and throws an error during boot times.
> (Why the issue was filed)

## Expected behavior
It should start :D
> (What you expected to happen)

## Actual behavior
The addon seems to start out fine, but it shockes on one of the init scripts:
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: InfluxDB
 Scalable datastore for metrics, events, and real-time analytics
-----------------------------------------------------------
 Add-on version: 3.7.0
 You are running the latest version of this add-on.
 System: HassOS 4.8  (amd64 / qemux86-64)
 Home Assistant Core: 0.110.4
 Home Assistant Supervisor: 225
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] create-users.sh: executing... 
[22:24:00] INFO: InfluxDB init process in progress...
/var/run/s6/etc/cont-init.d/create-users.sh: line 56: pgrep: command not found
[cont-init.d] create-users.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```
> (What actually happened)

## Steps to reproduce
Try to start a clean instance of the addon, I tried on 2 different systems.
> (How can someone else make/see it happen)

## Proposed changes
I am not sure, because I can't test older versions of the addon, but it might be related to the change from Ubuntu -> Debian base image? If my searching is correct, the `pgrep` command comes from the `procps` package, which is not installed by default in the Debian image.
> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
Ouch that is a bug... thanks for reporting that @Tommatheussen 👍

**Comment by gilmoth** [comment]
I'm having the same issue
dd-on: InfluxDB
 Scalable datastore for metrics, events, and real-time analytics
-----------------------------------------------------------
 Add-on version: 3.7.0
 You are running the latest version of this add-on.
 System: HassOS 3.13  (armv7 / raspberrypi4)
 Home Assistant Core: 0.110.4
 Home Assistant Supervisor: 225
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to TRACE
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] create-users.sh: executing... 
[22:48:23] TRACE: bashio::fs.file_exists: /data/secret
[22:48:26] INFO: InfluxDB init process in progress...
[22:48:31] INFO: InfluxDB init process in progress...
[22:48:36] INFO: InfluxDB init process in progress...
/var/run/s6/etc/cont-init.d/create-users.sh: line 56: pgrep: command not found
[cont-init.d] create-users.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.

**Comment by frenck** [comment]
Fixed in v3.7.1 of the add-on, which should be available any moment now.
